wnfs-wasm/src/fs/private/directory.rs

```diff
@@ -81,14 +81,13 @@ impl PrivateDirectory {
     ) -> JsResult<Promise> {
         let time = DateTime::<Utc>::from(time);
         let mut store = ForeignBlockStore(store);
+        let mut forest = Rc::clone(&forest.0);
```

Comment:
PrivateForest is owned. Can we do this instead?

```suggestion
        let mut forest = forest.0;
```